src/libprojectM/KeyHandler.cpp

```diff
@@ -120,25 +120,35 @@ void projectM::default_key_handler( projectMEvent event, projectMKeycode keycode
 		switch (keycode)
 		{
 		case PROJECTM_K_HOME:
-			if (renderer->showmenu) { // pageup only does something when the preset menu is active.
+			if (renderer->showmenu && !isTextInputActive()) { // pageup only does something when the preset menu is active.
```

Comment:
these could be all simplified to `if (isTextInputActive()) break;`

Reply:
done